package.json

```diff
@@ -13,7 +13,6 @@
     "type": "git",
     "url": "https://github.com/lynnaloo/oran.git"
   },
-  "pre-commit":["test"],
```

Comment:
Nice job!! I think @lynnaloo said your PR is next so she'll merge it in if she doesn't see any problems.